src/pages/SignUp/index.tsx

```diff
@@ -0,0 +1,98 @@
+import styled from '@emotion/styled';
+import { useState } from 'react';
+
+import KAKAO_LOGO from '@/assets/kakao_logo.svg';
+import { Button } from '@/components/common/Button';
+import { UnderlineTextField } from '@/components/common/Form/Input/UnderlineTextField';
+import { Spacing } from '@/components/common/layouts/Spacing';
+import { breakpoints } from '@/styles/variants';
+import { authSessionStorage } from '@/utils/storage';
+
+export const SignupPage = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleSignup = async () => {
+    if (!email || !password) {
+      alert('이메일과 비밀번호를 입력해주세요.');
+      return;
+    }
+
+    try {
+      const response = await fetch('/api/members/register', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ email, password }),
+      });
+
+      if (response.status === 201) {
+        const data = await response.json();
+        authSessionStorage.set(data.token);
+
+        const redirectUrl = `${window.location.origin}/`;
+        window.location.replace(redirectUrl);
```

Comment:
```suggestion
        window.location.replace(window.location.origin);
```

이렇게 작성하는 것과 동일할 것 같습니다~